www/src/components/blog-post-preview-item.js

```diff
@@ -56,15 +70,14 @@ class BlogPostPreviewItem extends React.Component {
               <Link
                 to={post.frontmatter.author.fields.slug}
                 css={{
-                  boxShadow: `none !important`,
-                  borderBottom: `0 !important`,
+                  color: `${colors.gatsby} !important`,
+                  fontSize: `102%`,
                   position: `relative`,
                   zIndex: 1,
                   "&&": {
-                    fontWeight: `normal`,
+                    fontWeight: `bold`,
```

Comment:
Please try `normal` here to give the link less weight—we don't want this to take away from the blog post title too much: 
![image](https://user-images.githubusercontent.com/21834/41290278-b2d454e8-6e4c-11e8-912a-a2d032c7fbb4.png)

Reply:
I mirrored these requested changes to the preview on post as well, thinking we would want agreement, but I split them into 2 commits so it would be easy to revert if you liked the bold in the post view. I will add a screenshot comparison in a sec.